### ISSUE TITLE
[Question] Any css or guideline to form fields

### ISSUE THREAD

**Issue by jrbenito** [LouisBarranqueiro/hexo-theme-tranquilpeak#255]
hi Louis,

Is there any general guideline for or CSS styling for form fields in this theme? 
Sorry, I could not locate any doc regarding this, if I lose it shame on me.

thanks

**Comment by LouisBarranqueiro** [comment]
Hey man, what do you want exactly? Do you wanna stylize form on all-* pages?

**Comment by jrbenito** [comment]
Hey!

Sorry man, I am versed on coding but almost blind for style.

I created a page with `hexo new page contact`, and put some html <form>... inside. To make things beautiful I need to blend the form with Tranquilpeak theme so I am wondering if is there any style already in place for forms (CSS classes) or do I need to write some custom CSS to make my ugly form seems to come from same forge as other elements ;)

**Comment by LouisBarranqueiro** [comment]
There is only this file : https://github.com/LouisBarranqueiro/hexo-theme-tranquilpeak/blob/master/source/_css/components/_form.scss

So put `form-control` class on input. But yeah, you will have to write some CSS. Read developer doc to build theme (speciality to build CSS) :)

**Comment by jrbenito** [comment]
That´s  what I had most fear you to answer :)

Fear not, let´s dive and learn some CSS. Thanks again man!

**Comment by LouisBarranqueiro** [comment]
If you got any questions, re-open this issue

Thanks for using this theme :)

**Comment by jrbenito** [comment]
Hi Louis,

Just to document for anyone reading this, I was able to todo my contact form as a page and a little customization in the theme layout as follows:

- Create a page for hold contact form and  thanks for the form submitted:
  `hexo  new page contact` creates contact/index.md and I put a contact/thanks.md by hand
- Put `form: true` in the front-matter of contact/index.md to show form (false will hide form for thanks page)
- Edit  layout/page.ejs:
```
<% if (page.path.indexOf('contact') > -1){ %>
       <%- partial('_partial/contact', {post: page}) %>
<% } else { %>
       <%- partial('_partial/post', {post: page}) %>
<% } %>
```
- Created _`partial/contact*` for holding contact specifics. File `_partial/contacts.ejs` is almost identical to `_partial/post.ejs` and includes `_partial/contact/form.ejs` that holds the HTML form. I also have a customized `_partial/contact/header.ejs` to get ridge of date below the title.

Works well, most of the style I inherited  this way and only fields and label need aditional CSS (what I am working on right now).

This form submits to formspree.io.
